lib/feedsme/index.js

```diff
@@ -54,153 +64,223 @@ function pkgName(pkg) {
  * @param {Fucntion} done Completion callback.
  * @api private
  */
-Feedsme.prototype.change = function change(env, data, done) {
-  var pkg = this.extractLatest(data);
-  var app = this.app;
-  //
-  // TODO: Figure out if we can get previous here or if we need to do a GET on
-  // artifactory which would kind of suck
-  //
-
-  var meta = {
-    name: pkg.name,
-    version: pkg.version,
-    env: env
-  };
+Feedsme.prototype.change = async function change(env, data) {
+  const { Dependent, DependentOf } = this.models;
+  const pkg = this.extractLatest(data);
+  const publish = env === 'dev';
```

Comment:
is that strictly true? Do we support rollback/dist-tag to older  in DEV?

Reply:
You are technically correct here. Rollback in DEV is not a highly requested feature. I would log an issue to support it but not block this work.

Reply:
@msluther this is a good callout. Technically we need to differentiate from a dist-tag and a publish since `dist-tag` can happen for dev. rollback is less likely and we can still handle it for DEV if we correctly make this differentiation.